tests/config/test_claude_md.sh

```
@@ -4,15 +4,15 @@
-    assert_file_exists "$ORIGINAL_DIR/CLAUDE.md" \
-        "CLAUDE.md should exist in repository root"
+    assert_file_exists "$ORIGINAL_DIR/system-configs/CLAUDE.md" \
+        "System CLAUDE.md should exist in system-configs directory"
 }
 
-# Test CLAUDE.md effectiveness criteria
+# Test CLAUDE.md effectiveness criteria - check system config for orchestration
 test_claude_md_effectiveness() {
-    local claude_file="$ORIGINAL_DIR/CLAUDE.md"
+    local claude_file="$ORIGINAL_DIR/system-configs/CLAUDE.md"
     local line_count=$(wc -l < "$claude_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)